pkg/image/api/v1/types.go

```diff
@@ -417,8 +417,10 @@ type ImageImportSpec struct {
 
 	// ImportPolicy is the policy controlling how the image is imported
 	ImportPolicy TagImportPolicy `json:"importPolicy,omitempty" protobuf:"bytes,3,opt,name=importPolicy"`
+	// ReferencePolicy defines how other components should consume the image
+	ReferencePolicy TagReferencePolicy `json:"referencePolicy,omitempty" protobuf:"bytes,4,opt,name=referencePolicy"`
 	// IncludeManifest determines if the manifest for each image is returned in the response
-	IncludeManifest bool `json:"includeManifest,omitempty" protobuf:"varint,4,opt,name=includeManifest"`
+	IncludeManifest bool `json:"includeManifest,omitempty" protobuf:"varint,5,opt,name=includeManifest"`
```

Comment:
Yo can't do this - you just broke the protobuf serialization.  *Never* change the tag number on a field.